### DISCUSSION TITLE
draw pictures using matplotlib

### DISCUSSION THREAD

**Discussion by xyysharon** [zwicker-group/py-pde#191] [Q&A]
Thanks for the great py-pde work. 
When I tried to draw pictures using matplotlib 3.5, there was a flash back. And then I switch to version 3.2, it works well. 
My code is as below:
![test](https://user-images.githubusercontent.com/57993404/146511889-eee1a79c-b1ba-4ece-b44f-2a9d661a52e3.png)
There is no error after execution, but the drawing window just flashes and exits.

**Comment by david-zwicker** [comment]
I could not reproduce the error. Could you please add additional information to [the issue](https://github.com/zwicker-group/py-pde/issues/192)?!

It would be best if you could post the output of the script `pde/scripts/show_environment.py`, so I get your exact versions of installed packages. Moreover, a more detailed error description might also help – what do you mean by a "flash back"?
